.github/actions/init_rust_job/action.yml

```diff
@@ -2,42 +2,46 @@ name: CLI rust job init
 description: Base steps for CLI jobs
 
 inputs:
-  cdn:
-    description: CDN to use when fetching assets
-    required: true
   platform:
     type: choice
-    description: Target platform to use when setting up rust
+    description: Target platform to use when installing nextest
     required: true
     options:
       - windows
       - macos
       - linux
-  assets_version:
-    description: Version of the assets to fetch from CDN
-    required: true
 
 runs:
   using: 'composite'
   steps:
-    - name: move CLI to root
```

Comment:
Why was this removed?
Some of the tests / tooling rely on this

Reply:
no longer needed